doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Authoring/DataSets/adding-data-sets-proc.adoc

```diff
@@ -0,0 +1,29 @@
+[id='adding-data-sets-proc_{context}']
+= Adding data sets
+
+You can create a data set to fetch data from an external data source and use that data for the reporting components.
+
+.Procedure
+. In {CENTRAL}, go to *Admin* -> *Data Sets*.
++
+The *Data Set Explorer* page opens.
+. Click *New Data Set* and select one of the following provider types:
++
+* *Bean:* Use to generate a data set from a Java class
+* *CSV:* Use to generate a data set from a remote or local CSV file
+* *SQL:* Use to generate a data set from an ANSI-SQL compliant database
+* *Elastic Search:* Use to generate a data set from Elastic Search nodes
+* *Prometheus:* Used to generate a data set using the Prometheus query
```

Comment:
There will be another new type of data set, called Kafka. At this moment, it is under review, but it will be in the 7.11 version of RHPAM/RHDM. Since it is not far from being included in the product, I think you could include it here as well.

Reply:
Thank you for mentioning the Kafka data set. The 7.11 epics are not yet created for the documentation. Also, it is not visible in the UI as well so I will add it once it is available and include it in another PR.
Thank you so much for pointing it out.  have taken note of it.